audit.txt

```diff
@@ -0,0 +1,2 @@
+17-05-2018 14:20:37 - Order ID: 109 - Employee: Chris Evans - Total Amount: R$20000.0
```

Comment:
this file must not be in git, you need to add it to .gitignore file

Reply:
ok